### ISSUE TITLE
Unable to find a specification for `MetalPerformanceShadersProxy`

### ISSUE THREAD

**Issue by simonbengtsson** [xmartlabs/Bender#65]
When running `pod install` with `pod 'MetalBender', :git => 'https://github.com/xmartlabs/Bender.git'` I get the following error in the terminal.

> [!] Unable to find a specification for `MetalPerformanceShadersProxy`

Looking at the cocoa pods specs it seams the `MetalPerformanceShadersProxy` were removed a few hours ago. Did I miss something?

![screen shot 2017-08-09 at 13 36 30](https://user-images.githubusercontent.com/3586691/29119810-e20e97aa-7d07-11e7-874d-b61b81589f12.png)

Cocoa Pods 1.3.1, Xcode 8.3.3, ios 10 with iphone 6

**Comment by bryant1410** [comment]
Yeah, we deleted yesterday to upload it as `MetalProxy`, as it involves not only `MetalPerformanceShaders` :sweat_smile: 

Today we will fix it, but as a workaround, you can just fork it, make MetalPerformanceShadersProxy in the podspec file point to the github repo (https://github.com/xmartlabs/MetalPerformanceShadersProxy) and then use that as a pod.

**Comment by simonbengtsson** [comment]
Sounds good! Tried to simply override the location of `MetalPerformanceShadersProxy` now which worked:

```
  pod 'MetalPerformanceShadersProxy', :git => 'https://github.com/xmartlabs/MetalPerformanceShadersProxy.git'
  pod 'MetalBender', :git => 'https://github.com/xmartlabs/Bender.git'
```

**Comment by bryant1410** [comment]
Good to know! Btw, we are planning to release a version of Bender to CocoaPods today :smile:

**Comment by bryant1410** [comment]
Ok, fixed by #66

Btw, `MetalPerformanceShadersProxy` still keeps its name.
